src/librustc_typeck/lib.rs

```diff
@@ -83,6 +86,7 @@ mod collect;
 mod constrained_generic_params;
 mod structured_errors;
 mod impl_wf_check;
+mod mem_categorization;
```

Comment:
Uh, this needs to be public too, Clippy can't use EUV without memcat

Reply:
In general we should be very careful about marking existing things as private

Reply:
Oh, I see, you want us to use Place from EUV, and Categorization isn't a thing anymore